openmls/src/group/core_group/test_external_init.rs

```diff
@@ -302,15 +343,30 @@ fn test_external_init_single_member_group(
     )
     .expect("An unexpected error occurred.");
 
-    // Have Alice export everything that Charly needs.
-    let pgs_encoded: Vec<u8> = group_alice
-        .export_public_group_state(backend, &alice_credential_bundle)
-        .expect("Error exporting PGS")
-        .tls_serialize_detached()
-        .expect("Error serializing PGS");
-    let verifiable_public_group_state =
-        VerifiablePublicGroupState::tls_deserialize(&mut pgs_encoded.as_slice())
-            .expect("Error deserializing PGS");
+    let group_info = {
```

Comment:
Can you add a negative test case where the signature is broken to make sure we check the signature?